src/main/java/io/github/bobocodebreskul/context/annotations/RequestBody.java

```diff
@@ -0,0 +1,15 @@
+package io.github.bobocodebreskul.context.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Applied to method parameters in controller method which specified with RequestMapping.
+ */
```

Comment:
Since it's expected to be used by clients it would be nice to have a usage example here.

Reply:
done